src/Redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Booksec = () => (
```

Comment:
- [ ] In this `Booksec` component, I think you still have to implement some changes as the previous reviewer requested because according to the project requirements you should have a component(Booksec) that renders 2 components which are Book(title, author) and CreateNewBook(form). Take a look at below screenshot, please 👇 
![Screenshot (135)](https://user-images.githubusercontent.com/67087664/149884833-2ae87f27-8e21-4704-a2e4-b7caa3d8feb6.png)

Reply:
The problem is that the project requirements you are showing are different from mine,
Here are the project requirements I have on my dashboard 👍 
**Bookstore: initialize project with components**

- Initialise React app.

- Add React Redux (npm install react-redux).

- Structure your application files using a "feature folder" approach and use the ducks pattern for your Redux files. Your files/folder structure could look like this:

...
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | configureStore.js
    | index.js

- The building blocks of your app should be set as re-usable components.

- Add React Router and set two <Route>s and <Link>s for the app's navigation:

- Books - the default view
Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)
Should have a form for adding a book (no functionality yet)

- Categories
Should display "Under construction" text only.

- Styling is not required at this point.

Reply:
I have talked to the previous reviewer about this issue, he understands and ask to resubmit. thank you